ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -896,6 +909,7 @@ let rules =
   ; upgrade_cluster_timeouts
   ; upgrade_secrets
   ; remove_legacy_ssl_support
+  ; empty_pool_uefi_certificates
```

Comment:
I believe this gets executed before calling `Xapi_host.write_uefi_certificates_to_disk` at startup